src/rupali_umagol/Assignment78.java

```diff
@@ -0,0 +1,56 @@
+/* *
+**
+***
+****
+
+1
+1 2
+1 2 3 
+1 2 3 4
+1 2 3 4 5
+1 2 3 4 5 6
+
+1
+2 3
+4 5 6
+7 8 9 10*/
+package rupali_umagol;
+
+public class Assignment78 {
+
+	void patternStar() {
+		for (int i = 1; i <= 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible